portal-web/test/functional/com/liferay/portalweb/macros/PageEditor.macro

```diff
@@ -311,6 +311,12 @@ definition {
 			value1 = "${commentReply}");
 	}
 
+	macro resolveComment {
```

Comment:
This is ok for now. We may need to rename this or create a new macro when resolving comments when the "Show Resolved Comments" checkbox is enabled. We should assert in that case that the reply button is disabled.

![image](https://user-images.githubusercontent.com/8635473/62916131-b2f53000-bdc9-11e9-9d01-8355402af159.png)